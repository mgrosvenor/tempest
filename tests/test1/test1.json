{
  "head":"##head.txt",
  "body":"This is the body",
  "long": "A multi-\nline string",
  "foot":"##foot.txt"
}
