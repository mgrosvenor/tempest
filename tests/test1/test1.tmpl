##head
$$body
$nosub
$$$$escaped
$${long}s
$${none}s
##foot
#nosub
####escaped
