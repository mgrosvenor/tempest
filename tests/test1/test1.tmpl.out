This is the header


Still the header

This is the body
$nosub
$$escaped
A multi-
line strings
$${none}s
This is the footer

#nosub
##escaped
